includes/class.llms.query.user.postmeta.php

```diff
@@ -87,7 +87,8 @@ public function get_metas() {
 	 *
 	 * @return   void
 	 * @since    3.15.0
-	 * @version  3.15.0
+	 * @since    [version] Added 'Favorites' event.
+	 * @version  [version]
```

Comment:
```suggestion
	 * @since [version] Added 'Favorites' event.
```
we don't add the `version` tag to the function/methods anymore, as per our documentation standards.